Flabbergasted-Flamingos/bangazon-api
# Cd payment categories

# Description
This pull request covers the addition of a Payment Category to our api

## Number of Fixes


## Related Ticket(s)
First part of ticket #4

## Problem to Solve
Added payment category to db.

## Proposed Changes
Added payment category model, serializer, view, and url.

## Expected Behavior


## Steps to Test Solution

1. Pull down my branch:
```
git fetch --all
git checkout cd-Payment
```
2. Make sure you are in the directory bangazon-api

3.  If you have an existing db.sqlite3, delete it.

3. Migrate:
```
python manage.py api
```
4. Run server:
```
python manage.py runserver
```

5. Go to the following url: http://127.0.0.1:8000/api/paymentcategories/

6. Add data

7. Go to the url: http://127.0.0.1:8000/api/paymentcategories/1 and you should see the first piece of data that you added. Then you can try /2, /3, etc...

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
